test_scripts/Polices/External_UCS/External_Consent_OFF/024_ATF_Policies_External_Consent_OFF_many_entities_pairs_in_one_group_01.lua

```diff
@@ -0,0 +1,223 @@
+--------------------------------------Requirement summary---------------------------------------------
```

Comment:
@vvvakulenko Rejected, it is easier to define them on top then to update all functions that use them